src/generators/minecraft-mutant-character/MinecraftMutantCharacterGenerator.res

```diff
@@ -0,0 +1,507 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "minecraft-mutant-character"
+
+let name = "Mutant Character"
+let video: Generator.videoDef = {url: "https://www.youtube.com/embed/DVzumgRinjY"}
+let thumbnail: Generator.thumnbnailDef = {
+  url: Generator.requireImage("./thumbnail/thumbnail.jpeg"),
+}
+let images: array<Generator.imageDef> = [
+  {id: "Page1", url: requireImage("Page1.png")},
+  {id: "Page2", url: requireImage("Page2.png")},
+  {id: "Page3", url: requireImage("Page3.png")},
+  {id: "Page4", url: requireImage("Page4.png")},
+  {id: "Hole1", url: requireImage("Hole1.png")},
+  {id: "Hole2", url: requireImage("Hole2.png")},
+  {id: "Hole3", url: requireImage("Hole3.png")},
+  
+]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin",
+    url: requireTexture("Skin.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  }
+]
+
+let script = () => {
+  
+Generator.defineTextureInput("Skin", {
+        standardWidth: 64,
+        standardHeight: 64,
+        choices: []
+      });
```

Comment:
Formatting needs to be fixed in this file

Reply:
Done